lib/routes/account.js

```diff
@@ -544,20 +541,19 @@ module.exports = function (
         }
 
         function sendVerifyAccountEmail() {
-          // For legacy clients, behavior is to not send an email to verify their account
-          // and have the requestor send it.
-          var sendEmailIfUnverified = false
-
-          // If a value was passed, use that instead
-          if (request.payload.sendEmailIfUnverified !== undefined) {
-            sendEmailIfUnverified = request.payload.sendEmailIfUnverified
-          }
+          // Delegate sending emails for unverified users to auth-server.
+          // Will be removed once all clients have been updated not to send verify emails.
+          // https://github.com/mozilla/fxa-auth-server/issues/1325
+          var shouldSendVerifyAccountEmail = requestHelper.shouldSendVerifyAccountEmail(request)
+          emailSent = false
 
-          var shouldSendVerifyAccountEmail = sendEmailIfUnverified && !emailRecord.emailVerified
           if (shouldSendVerifyAccountEmail) {
```

Comment:
Good catch, there were some gaps in the testing that didn't check this. Updated!